components/api_server/pyproject.toml

```diff
@@ -36,7 +35,6 @@ optional-dependencies.tools = [
 ]
 
 [tool.ruff]
-target-version = "py312"
```

Comment:
Is there no added benefit to specifying a target version?

Reply:
It's (no longer?) needed: "If you're already using a pyproject.toml file, we recommend project.requires-python instead" (https://docs.astral.sh/ruff/settings/#target-version).